src/components/data/toolbar/Toolbar.js

```diff
@@ -188,6 +238,32 @@ function DataToolbar(props) {
                             refreshListing();
                         }}
                     />
+                    <ConfirmationDialog
+                        baseId={baseId}
+                        open={deleteConfirmOpen}
+                        onClose={() => {
+                            setDeleteConfirmOpen(false);
+                        }}
+                        title={t("Delete")}
+                        contentText={t("deleteConfirmation")}
+                        onConfirm={() => {
+                            onDeleteSelected();
+                            setDeleteConfirmOpen(false);
+                        }}
+                    />
+                    <ConfirmationDialog
+                        baseId={baseId}
```

Comment:
In this context, `baseId` is actually the ID for the entire dialog.  So in this case, this dialog and the one above should have their own, new IDs, not the same ID as the toolbar. 
Maybe the `baseId` prop in `ConfirmationDialog` can be renamed to `dialogId` to make it less confusing in another PR.